CODEOWNERS

```diff
@@ -1,17 +1,17 @@
 acct.bionimbus.org/manifest.json @ac3eb @uc-cdis/planx-qa
 caninedc.org/manifest.json @trevars @uc-cdis/planx-qa
-data.bloodpac.org/manifest.json @fantix @uc-cdis/planx-qa
+data.bloodpac.org/manifest.json @trevars @uc-cdis/planx-qa
 genomel.bionimbus.org/manifest.json @trevars @uc-cdis/planx-qa
 
-data.braincommons.org/manifest.json @dgrantkeane @uc-cdis/planx-qa
-dataprep.braincommons.org/manifest.json @dgrantkeane @uc-cdis/planx-qa
-staging.braincommons.org/manifest.json @dgrantkeane @uc-cdis/planx-qa
-gen3testing.braincommons.org/manifest.json @dgrantkeane @uc-cdis/planx-qa
-tbi.datacommons.io/manifest.json @dgrantkeane @uc-cdis/planx-qa
-gen3-neuro.datacommons.io/manifest.json @dgrantkeane @uc-cdis/planx-qa
-va-testing.datacommons.io/manifest.json @dgrantkeane @uc-cdis/planx-qa
-va.datacommons.io/manifest.json @dgrantkeane @uc-cdis/planx-qa
-vpodc.org/manifest.json @dgrantkeane @uc-cdis/planx-qa
+data.braincommons.org/manifest.json @jingh8 @uc-cdis/planx-qa
+dataprep.braincommons.org/manifest.json @jingh8 @uc-cdis/planx-qa
+staging.braincommons.org/manifest.json @jingh8 @uc-cdis/planx-qa
+gen3testing.braincommons.org/manifest.json @jingh8 @uc-cdis/planx-qa
+tbi.datacommons.io/manifest.json @jingh8 @uc-cdis/planx-qa
+gen3-neuro.datacommons.io/manifest.json @jingh8 @uc-cdis/planx-qa
+va-testing.datacommons.io/manifest.json @trevars @uc-cdis/planx-qa
+va.datacommons.io/manifest.json @trevars @uc-cdis/planx-qa
+vpodc.org/manifest.json @trevars @uc-cdis/planx-qa
 
 gen3.theanvil.io/manifest.json @GARUPP-zz @uc-cdis/planx-qa
```

Comment:
hey @jingh8  can you update the the CODEOWNER for anvil envs as well in this PR? you can change it to `ac3eb` Alessandro Culotti